packages/internal-test-helpers/lib/maybe-expect-deprecation.ts

```diff
@@ -0,0 +1,16 @@
+import { ExpectDeprecationFunc } from './ember-dev/deprecation';
```

Comment:
Have you tried just doing `import './ember-dev/deprecation';`? I _think_ that should cause the compiler to "see" the `declare global { ... }` block in that file, which should make `expectDeprecation` available here.